README.md

```diff
@@ -117,11 +117,23 @@ kubectl port-forward service/repo 9092:80 --namespace workshop-samples
 export MAVEN_USER=desdp
 export MAVEN_PASSWORD=password
 ```
--   Build and publish your application JAR file.
+-   Build and publish your application JAR file. Make sure to replace the `release name` and `application values file` with appropriate values.
 ```
 ./gradlew publish
-helm upgrade --install --timeout 600s jsonreader \
---wait --namespace workshop-samples charts
+helm upgrade --install --timeout 600s --wait \
+    <Release Name> \
+    charts/workshop-samples \
```

Comment:
Is `workshop-samples` a good name for this chart? This chart is general-purpose but only for Flink jobs. Spark and other types of apps will need to use different charts. So you might want to rename the chart to `flink-app`, `workshop-samples-flink-app`, or `sdp-flink-app`.

Reply:
Update the chart using `flink-app`